docs/source/changelog.rst

```diff
@@ -14,6 +14,58 @@ For upcoming changes and next releases, see our `milestones <https://github.com/
 For our long-term plan, see our :ref:`roadmap`.
 
 
+Version 0.12.1 - 2022-06-28
+---------------------------
+
+Enhancements
+~~~~~~~~~~~~
+
+* [ENH] test univariate forecasting with ``pd.DataFrame`` input and longer ``fh``  (:pr:`2581`) :user:`fkiraly`
+* [ENH] subsetting transforms (:pr:`2831`) :user:`fkiraly`
+* [ENH] A new `ReconcilerForecaster` estimator for reconciling forecasts using base model residuals  (:pr:`2830`) :user:`ciaran-g`
+* [ENH] Lagging transformer (:pr:`2783`) :user:`fkiraly`
+* [ENH] Adding | dunder for MultiplexTransformer (:pr:`2810`) :user:`miraep8`
+
+
+Documentation
+~~~~~~~~~~~~~
+
+* [DOC] add missing Sajay references (:pr:`2800`) :user:`fkiraly`
+* [DOC] add missing TonyBagnall to contributors of 0.12.0 in changelog (:pr:`2803`) :user:`fkiraly`
+* [DOC] adds solution to "no matches found" to troubleshoot section of install guide (:pr:`2786`) :user:`AurumnPegasus`
+* [DOC] cleaning up transformer API reference (:pr:`2818`) :user:`fkiraly`
+* [DOC] team update: remove Tony from CC (:pr:`2794`) :user:`fkiraly`
+* [EHH] more informative error messages on input format (:pr:`2824`) :user:`fkiraly`
```

Comment:
typo, and should not be documentation?

Reply:
not sure it's your PR so let me know which section does it fit